app/react-native/readme.md

```diff
@@ -72,6 +72,21 @@ For RN apps:
 
 Once your app is started, changing the selected story in web browser will update the story displayed within your mobile app.
 
+## Using Haul-cli
+
+[Haul](https://github.com/callstack-io/haul) is an alternative to the react-native packager and has several advantages in that it allows you to define your own loaders, and handles symlinks better.
+
+If you want to use haul instead of the react-native packager, modify they storybook npm script too:
```

Comment:
I'm so bad with spelling. 

I need some kind of ai to just point this kinda stuff out. If only there were some kind of build in tools...